### DISCUSSION TITLE
Which websites are used in Online Mode ?

### DISCUSSION THREAD

**Discussion by sinux-l5d** [aquasecurity/trivy-operator#2173] [Help & Support] [answered]
Hello everyone,

I'm currently using offline mode to scan images, but I'll eventually have to switch to online mode. 

But I can't find an official list of websites accessed during scanning. In PR https://github.com/aquasecurity/trivy-operator/pull/677, I see it uses _search.maven.org_, but what else ?

I would really appreciate it if someone can share his experience 😄

**Comment by chen-keinan** [comment, marked as answer]
@sinux-l5d 

in general :
- search.maven.org
- github.com/aquasecurity

**Reply by sinux-l5d** [reply]
Thanks @chen-keinan, that's pretty straightforward to add to our filtering rules :D
